docs/updating-schema.md

```diff
@@ -2,20 +2,26 @@ HHAST is closely coupled to Hack's syntax tree, so if the schema of `hh_parse`
 changes, HHAST needs updating.
 
 1. Get a github codespace or use vscode devcontainers. If a codespace, pick the biggest one you can for this (in general, the smallest are good enough, but here more power is useful).
-2. Edit `/etc/apt/sources.list` - remove `-4.x` suffix from dl.hhvm.com repository
-3. `apt update; apt remove -y hhvm; apt install -y hhvm-nightly` (or `apt install hhvm` for latest stable release)
-4. `cd /workspaces; git clone --depth 1 https://github.com/facebook/hhvm.git`
-5. `cd hhast`
-6. `bin/update-schema --hhvm-path=/workspaces/hhvm`
-7. wait 5-15 minutes as it parses and analyzes the hack/hhvm unit tests to infer valid node relationships
-8. `git diff codegen/schema.json codegen/inferred_relationships.hack` - are the schema changes going to change how code that is valid on both versions is represented?
-9. Adjust `src/__Private/is_compatible_schema_version.hack`, following the instructions in the code comments
-10. Fix `hh_client`, `bin/hhast-lint`, and `vendor/bin/hacktest tests/` errors
-11. If the changes are not backwards compatible, update the minimum versions in composer.json and .github/workflows/ci.yaml
-12. Create pull request
-13. Once merged and CI is green on main branch, tag a new release, matching the new minimum HHVM version
+1. Execute the following command
+   ```
+   sed -i -E 's/deb https\:\/\/dl\.hhvm\.com\/ubuntu (\w+)-4.[0-9]+main/deb https:\/\/dl.hhvm.com\/ubuntu \1 main/' /etc/apt/sources.list &&
```

Comment:
This is not working when I try it. IIUC there needs to be a space after the [0-9]+ and before the word 'main' in order to make this work correctly.